aip/general/0193.md

```diff
@@ -62,11 +62,90 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
+### Error messages
 
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+The [`google.rpc.Status`][Status] error payload **should** contain two
+error messages.
+
+1. The payload **must** contain an error message in the `message` field
+   found at the root of [`Status`][Status].
+1. The payload **should** contain
+   [`google.rpc.LocalizedMessage`][LocalizedMessage] in `details` on
+   `Status`. This is the location of the second error message.
+
+The error message on `Status` should,
```

Comment:
Is this a requirement word "should" or best practices "should"? 

Perhaps this:
> The error message `Status` is most effective when it,
>
> - is developer-facing, ...
> - explains the error...

OR make it a `**should**`